app/models/validators/message_drafts_import_validator.rb

```diff
@@ -17,12 +17,16 @@ def validate(record)
             else
               record.errors.add(:drafts, "Disallowed draft subfolder")
             end
+          elsif draft_subfolder_name.start_with?('.')
+            # noop
           else
             record.errors.add(:drafts, "Unknown signature status, files must be inside a folder.")
           end
         end
       elsif Utils.csv?(import_entry_name)
         # noop
+      elsif import_entry_name.start_with?('.')
+        # noop
```

Comment:
Pridala som tu podmienku, nech sa ignoruju `.` priecinky, lebo v opacnom pripade to robilo problemy napr. so ZIPkami z mac OS, kvoli par takymto priecinkom navyse.

Reply:
hej, podľa mňa good enough workaround. Ale spomeňme to potom niekde v dokumentácii, aby ľudia vedeli, že nemajú vyrábať také priečinky.